src/main/java/org/broadinstitute/hellbender/engine/ReferenceDataSource.java

```diff
@@ -99,19 +67,9 @@ public ReferenceSequence queryAndPrefetch( final String contig, final long start
      *
      * @return SAMSequenceDictionary for this reference
      */
-    public SAMSequenceDictionary getSequenceDictionary() {
-        return reference.getSequenceDictionary();
-    }
+    public SAMSequenceDictionary getSequenceDictionary();
+
+    @Override
+    public void close();
```

Comment:
This interface extends `AutoCloseable` already, so you don't need to declare a `public void close();` here.